src/qt/bantablemodel.cpp

```diff
@@ -6,11 +6,10 @@
 
 #include <interfaces/node.h>
 #include <net_types.h> // For banmap_t
-#include <qt/clientmodel.h>
 
 #include <utility>
 
-#include <QDebug>
+#include <QDateTime>
```

Comment:
nit: while here, `<QVariant>` and `<QModelIndex>` could also be added.

Reply:
Done, and pushed to #18036.